netkan/netkan/scheduler.py

```diff
@@ -72,29 +72,21 @@ def can_schedule(self, max_queued, dev=False):
             #       around 40 credits, with an accrue rate of 24/hr. So running every
             #       2 hours should see using just a touch less than we gain in that
             #       time period.
-            credits = 0
+            creds = 0
             try:
```

Comment:
Same as previous comment regarding shortening of the variable.

Reply:
![image](https://user-images.githubusercontent.com/1559108/67298591-19c15480-f4b1-11e9-981e-c0802cf0e77a.png)

https://docs.python.org/2/library/constants.html#credits

Reply:
Aha, didn't know credits was a constant, good catch. Though for reference, we don't care about python2, but it is a constant in 3 as well.